examples/generative/text_generation_gpt.py

```diff
@@ -193,19 +206,10 @@ def preprocess(inputs):
 # Output.
 outputs = keras.layers.Dense(VOCAB_SIZE)(x)
 model = keras.Model(inputs=inputs, outputs=outputs)
-loss_fn = tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True)
+loss_fn = keras.losses.SparseCategoricalCrossentropy(from_logits=True)
 perplexity = keras_nlp.metrics.Perplexity(from_logits=True, mask_token_id=0)
 model.compile(optimizer="adam", loss=loss_fn, metrics=[perplexity])
 
-"""
-Let's take a look at our model summary - a large majority of the
```

Comment:
Why did we remove this section?

Reply:
When I run a generation script, it simply fails saying that console width is not enough to print summary.

Reply:
Hmm, probably not a reason to cut this section. Do you have the error handy?